RFC/src/RFC-0201_TariScript.md

```diff
@@ -370,6 +370,8 @@ pub struct BlockHeader {
     pub total_script_offset: BlindingFactor,
 }
 ```
+One important distinction to make is that the coinbase utxo does not count towards the _script offset_. This is because the coinbase UTXO already has special rules accompanying it and it has no input. Thus we cant generate a \\( \so \\) for a coinbase transaction.
```

Comment:
**NIT**
Consider: Rather use "_Thus we **cannot** generate a ..._" than "_Thus we **cant** generate a ..._"